css/_layout.scss

```diff
@@ -7,35 +7,86 @@
 .content-align-center {
   display: flex;
   align-items: center;
+  justify-content: center;
+}
+.content-align-center--logo {
+  @include media-breakpoint-up(lg) {
+    justify-content: space-between;
+  }
+}
+.content-align-center--info {
+  @include media-breakpoint-up(sm) {
+    justify-content: space-between;
+  }
+}
+.content-align-center--info-spec {
+  @include media-breakpoint-up(sm) {
+    justify-content: flex-start;
+  }
+}
```

Comment:
You don't need all of this modifiers.
If you need to modify logo you should create modifier for logo)